rs/tests/os_qualification/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "os_qualification"
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+description.workspace = true
+documentation.workspace = true
+
+[dependencies]
+ic-tests = { path = "../" }
```

Comment:
I do import in `rs/tests/dre/utils/steps/xnet.rs`, a dep to messaging

Reply:
Ah darn you're right. Any chance you can move that dep out of rs/tests/src? That will bring us another step closer to  [IDX-3144](https://dfinity.atlassian.net/browse/IDX-3144).

Reply:
I'd rather messaging guys do it because I'm pretty new to the testing code and we don't own that. If they agree, in another PR, I am willing to do it. Trying to keep the scope of this PR as small as possible

Reply:
@derlerd-dfinity is that something your team can pick up? We really want to get rid of the whole `rs/tests/src` crate since it's a drag on CI and when developing system-tests locally.